sources/mongodb/helpers.py

```diff
@@ -235,9 +237,53 @@ def convert_mongo_objs(value: Any) -> Any:
         return str(value)
     if isinstance(value, _datetime.datetime):
         return ensure_pendulum_datetime(value)
+    if isinstance(value, Regex):
+        return value.try_compile().pattern
+    if isinstance(value, Timestamp):
+        date = value.as_datetime()
+        date = date.replace(tzinfo=timezone("UTC"))
```

Comment:
why `replace` is needed? this just forces UTC over any timezone that was there without conversion.
just run it via `ensure_pendulum_datetime`, it will add UTC timezone if necessary

Reply:
Done